src/falcon/mod.rs

```diff
@@ -0,0 +1,24 @@
+mod ffi;
+mod hash;
+mod keys;
+mod polynomial;
+mod signature;
+
+pub use keys::{KeyPair, PublicKey, SecretKey};
+pub use polynomial::Polynomial;
+pub use signature::Signature;
+
+// CONSTANTS
+// ================================================================================================
+
+const MODULUS: u16 = 12289;
+const MODULUS_MINUS_1_OVER_TWO: u16 = 6144;
+
+const LOG_N: usize = 9;
+const N: usize = 512;
```

Comment:
nit: could we give more expressive names to these constants? (and probably add some comments).

Reply:
Sure!